.github/workflows/vale.yml

```diff
@@ -3,7 +3,6 @@ name: Vale
 on:
   pull_request:
     paths:
-      - 'guides/common/**.adoc'
```

Comment:
This means it won't trigger on `adoc` files anymore, only when `valie.yml` is changed. I don't think that's what you intended. Instead I think we should add other vale related files to this list.

Reply:
I added an error .adoc file after reverting this line, but Vale did not check the new file.

Reply:
I see the problem. Vale is only going to report actual errors. I'm going to raise the minimum report level from "error" to "warning" to see how that goes.

Reply:
Works now! "warning" level was not a success, so I reverted to "error" level reporting and made sure I had real errors.